developer/testscripts/aatest_ds000114_fmri_localpipeline.m

```diff
@@ -0,0 +1,121 @@
+function aatest_ds000114_fmri_localpipeline(parameterfile, deleteprevious,wheretoprocess)
+
+% This script runs a local pipeline connection to first pre-process the study,
+% and then does modelling. It is the same script as ds000114_fmri except that
+% it uses a pipeline connection to use preprocessed results for modelling.
+% The script also uses symlinks to load stream files to the new modules instead of copying.
+
+% -------------------------------------------------------------------------
+% init
+% -------------------------------------------------------------------------
+
+aap = aa_test_inittest([mfilename('fullpath') '_prep'], parameterfile, deleteprevious, wheretoprocess);
+% -------------------------------------------------------------------------
+% First pre-process the data.
+% -------------------------------------------------------------------------
+
+aap.options.autoidentifystructural_choosefirst = 1;
+aap.options.autoidentifystructural_chooselast = 0;
+
+aap.options.remotesymlinks = 1; % Use symlinks instead of copying
+aap.options.NIFTI4D = 1;
+aap.acq_details.numdummies = 4;	
+aap.acq_details.numdummies = 1;
+aap.acq_details.input.correctEVfordummies = 1;
+
+aap.tasksettings.aamod_segment8.writenormimg = 0;
+aap.tasksettings.aamod_segment8.samp = 2;
+aap.tasksettings.aamod_slicetiming.autodetectSO = 1;
+aap.tasksettings.aamod_slicetiming.refslice = 16;
+aap.tasksettings.aamod_smooth.FWHM = 5;
+
+prep_dir = fullfile(aap.acq_details.root,aap.directory_conventions.analysisid);
```

Comment:
This should probably be
`prep_dir = fullfile(aap.acq_details.root,[aap.directory_conventions.analysisid '_prep']);`

Reply:
Thanks!

This prep_dir had worked when I tried it with the previous way, but this makes more sense so let me see what CI says about it.

Reply:
I was wrong. The main issue seems that the `aap.directory_conventions.analysisid` is not correctly set, and it stays `aatest_ds000114_fmri` for both pipelines. Therefore,  the second pipeline removes the first one.

Reply:
I forgot to mention that the testscripts uses a specific format aatest_<dataset>_<usecase>. As you can see, the fields MUST NOT contain '_'. So you should rename the files as "aatest_ds000114_fmrisymlinks", "aatest_ds000114_fmrilocalpipelineprep", and "aatest_ds000114_fmrilocalpipelinemodel".

Reply:
> I was wrong. The main issue seems that the `aap.directory_conventions.analysisid` is not correctly set, and it stays `aatest_ds000114_fmri` for both pipelines. Therefore, the second pipeline removes the first one.

Regarding this what is the solution? Shall I revert it to the old way?